src/main/java/org/runejs/client/MovedStatics.java

```diff
@@ -1633,71 +1634,68 @@ public static void manageTextInputs() {
 	                    SceneCluster.packetBuffer.putByte(-1 + ChatBox.chatboxInput.length());
 	                    SceneCluster.packetBuffer.putString(ChatBox.chatboxInput.substring(2));
 	                } else {
-	                    int i = 0;
+	                    int chatShapeEffect = 0;
 	                    String class1 = ChatBox.chatboxInput.toLowerCase();
-	                    int i_11_ = 0;
+	                    int chatColorEffect = 0;
 	                    if(class1.startsWith(English.effectYellow)) {
-	                        i_11_ = 0;
+	                        chatColorEffect = 0;
 	                        ChatBox.chatboxInput = ChatBox.chatboxInput.substring(English.effectYellow.length());
 	                    } else if(class1.startsWith(English.effectRed)) {
 	                        ChatBox.chatboxInput = ChatBox.chatboxInput.substring(English.effectRed.length());
-	                        i_11_ = 1;
+	                        chatColorEffect = 1;
```

Comment:
Could these maybe be replaced with an enum?

Reply:
good idea @Promises, will get that seen to

Reply:
@Promises Feedback addressed, moved them all into enums